assignments/arrays.js

```diff
@@ -63,32 +63,69 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+    
+var newCar = [];
+    
+for (let i = 0; i < inventory.length; i++) {
+
+    if (inventory[i].id === (33)) {
+
+        //newCar.push(inventory[i].car_year,inventory[i].car_make,inventory[i].car_model);
+        newCar.push(inventory[i])
+    }
+}
+
+console.log(`Car ${newCar[0].id} is a ${newCar[0].car_year} ${newCar[0].car_make} ${newCar[0].car_model}`);
+
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+
+const lastCar = inventory.length;
+console.log (inventory[lastCar-1]);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+
+for (let i = 0; i < inventory.length; i++){
+    carModels.sort().push(inventory[i].car_model);   
```

Comment:
I would recommend not sorting in the for loop. This is inefficient. Sort once outside of the for loop

Reply:
sorted in my console.log instead of in the for loop